docs/getting-started/install-resoto/kubernetes.md

```diff
@@ -53,37 +40,27 @@ These instructions were tested with version 1.2.15 of the operator.
 
 :::
 
-Then, wait until the ArangoDB deployment is ready:
+### Optional: adjust configuration via a Helm Values File
 
-```bash
-$ kubectl wait --for=condition=ready arangodeployment/single-server
-```
+It is possible to adjust the configuration of Resoto installation using a Helm values file.
 
-### Create Helm Values File
+Please find the list of all [`someengineering/helm-chart` GitHub repository](https://github.com/someengineering/helm-charts/tree/main/someengineering/resoto#values).
 
-```yaml title="resoto-values.yaml"
-resotocore:
-  graphdb:
-    server: http://single-server:8529
-```
+To override any value, please create the file `resoto-values.yaml` and define the values there.
 
-This is the minimum configuration, which points to an empty ArangoDB database with default username and password.
-
-The installation will create a separate database and password and secure the database installation with a generated password. You can find the generated database password in the secret `arango-user`.
+```yaml title="resoto-values.yaml example file"
+arangoDB.enabled: false
+```
 
-See the [`someengineering/helm-chart` GitHub repository](https://github.com/someengineering/helm-charts/tree/main/charts/resoto) for a list of configurable values.
+This step is optional. By default, a single ArangoDB database is created and secured by a generated password. You can find the generated database password in the secret `arango-user`.
 
 ### Install Helm Chart
```

Comment:
Can we actually make two section around:

- default installation
- Customised install

This would be more easier to read rather than mentioning things around in between.

Reply:
Like your proposal. Added a tab group for the 2 sections.